tests/integration/test_models/standard_output_tests.py

```diff
@@ -315,20 +324,23 @@ def test_concentration_limits(self):
     def test_conservation(self):
         """Test amount of lithium stored across all particles and in SEI layers is
         constant."""
-        L_n = self.param["Negative electrode thickness [m]"]
-        L_p = self.param["Positive electrode thickness [m]"]
-        L_y = self.param["Electrode width [m]"]
-        L_z = self.param["Electrode height [m]"]
-        A = L_y * L_z
-
         self.c_s_tot = (
             self.c_s_n_tot(self.solution.t)
             + self.c_s_p_tot(self.solution.t)
-            + self.n_SEI_n_av(self.solution.t) * L_n * A
-            + self.n_SEI_p_av(self.solution.t) * L_p * A
+            + self.c_SEI_n_tot(self.solution.t)
+            + self.c_SEI_p_tot(self.solution.t)
+            + self.c_Li_n_tot(self.solution.t)
+            + self.c_Li_p_tot(self.solution.t)
         )
         diff = (self.c_s_tot[1:] - self.c_s_tot[:-1]) / self.c_s_tot[:-1]
-        np.testing.assert_array_almost_equal(diff, 0)
+        if "profile" in self.model.options["particle"]:
+            np.testing.assert_array_almost_equal(diff, 0, decimal=10)
+        elif self.model.options["surface form"] == "differential":
+            np.testing.assert_array_almost_equal(diff, 0, decimal=10)
+        elif self.model.options["SEI"] == "ec reaction limited":
```

Comment:
shouldn't this be fine up to 1e-15 now that you are using the ""Loss of lithium to negative electrode SEI [mol]" variable?

Reply:
Now it is of the order of 1e-13 (it went down from something like 1e-9) so it is a significant improvement.